sonar-project.properties

```diff
@@ -0,0 +1,2 @@
+sonar.projectKey=ccd-case-activity-web
+sonar.exclusions=app/redis/redis-client.js
```

Comment:
why exclusion of redis client?

Reply:
that's the only one failing Sonar scan for now

Reply:
👍